ecommerce/models.py

```diff
@@ -68,6 +68,8 @@ class Order(AuditableModel):
     modified_at = DateTimeField(auto_now=True)
     total_price_paid = DecimalField(decimal_places=2, max_digits=20)
 
+    reference_number = CharField(null=True, unique=True, max_length=255)
```

Comment:
Is this something we might want to add to Django Admin's [display list of Order model](https://github.com/mitodl/micromasters/blob/master/ecommerce/admin.py#L41).

Reply:
I think for now we'll leave it to keep the changeset down and we haven't needed it yet.